src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1481,8 +1481,13 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
                 throw new LogicException('The Validation middleware is only available when the Validator component is installed and enabled. Try running "composer require symfony/validator".');
             }
 
+            $tagAttributes = array('name' => $name);
+            if (isset($bus['decorator_class'])) {
+                $tagAttributes['decorator_class'] = $bus['decorator_class'];
```

Comment:
should it fail if the class does not implement `MessageBusInterface` at least? Perhaps give a complete hint about what is needed for using this option